bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+Therefore, this BSIP defines a process for BitAssets management to ensure protocol security remains the singular focus of the block producer role, and give BitAssets the best chance for long-term viability by nominating a new custodian to quickly react to market conditions.  Furthermore, the portion of BTS collateral below the full valuation of a debt asset, as defined by its price feed, that is borrowed from the blockchain, shall not contribute to the voting power of the debtor.
+
+# Motivation
+Block production is an essential component of the BitShares platform itself, and as such should be expected to take an agnostic and neutral view towards the products deployed within the platform. Therefore, this BSIP will define a process for BitAsset management to ensure protocol security remains the singular focus of the block producer role and give BitAssets the best chance for long-term stability by nominating a new custodian to quickly react to market conditions. In order for the distributed consensus mechanism of DPOS to achieve adequate decentralization, it is imperative that block producers maintain strict independence from the politics of individual products deployed on the platform. 
+
+Governance is an important component of the core BTS token and effective application of governance throughout the BitShares protocol is imperative to the long-term success of the project.  A number of BSIPs have addressed the shortcomings of the existing vote calculation mechanism. This BSIP changes vote calculation which currently confers governance rights which are intended for assignment to the protocol.
+
+In short, this BSIP will put the BitAssets under the control of a designated user account. With this change, the newly appointed BitAssets manager will be able to independently maintain the parameters of their derivative asset without interference from the platform’s block producers and committee members. At the same time, the platform itself and its committee will not be unduly influenced by the demands of private product management. An advantage gained by this is that the BitAsset manager will be able to react appropriately to rapidly changing market conditions, as may be required to properly maintain a successful derivative asset.
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a directly appointed multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
```

Comment:
They're owned by the committee and fed by the witnesses. Ideally change requests would be sent to the committee, who would then issue agreed upon changes to the price feed publishers directly, instead of BSIPs attempting to directly influence/coerce witnesses to make changes (potentially against the wishes of the committee & to the detriment of network stability).

Not long ago I requested the committee to change the max price feed publisher quantity to 25, this was done via bitsharestalk after which a proposal was created and voted upon by the committee. This didn't require a BSIP, nor should bitasset settings/policies require BSIP votes, instead the committee could vote on a committee proposal for changing the asset (voting on proposed asset description to indicate to feeders the economic policies).

Why doesn't cn-vote join the committee for influence, rather than request full control?

Reply:
This BSIP is not the result of a request from cn-vote. Also see my remarks [below](https://github.com/bitshares/bsips/pull/238#issuecomment-540940222).

Reply:
Replaced cn-vote